src/Kunstmaan/AdminListBundle/Controller/AdminListController.php

```diff
@@ -28,6 +28,31 @@
  */
 abstract class AdminListController extends Controller
 {
+    /**
+     * {@inheritdoc}
+     *
+     * @deprecated
+     */
+    protected function get($id)
+    {
+        @trigger_error('Getting services directly from the container is deprecated in 5.1 and will be removed in 6.0. Register your controllers as services and inject the necessary dependencies.', E_USER_DEPRECATED);
```

Comment:
Context needed: in *KunstmaanBundle* 5.1

Reply:
Good catch, fixed!